src/query/frontend/opencypher/grammar/MemgraphCypherLexer.g4

```diff
@@ -67,6 +67,8 @@ HEADER              : H E A D E R ;
 IDENTIFIED          : I D E N T I F I E D ;
 IGNORE              : I G N O R E ;
 ISOLATION           : I S O L A T I O N ;
+IN_MEMORY_ANALYTICAL    : I N UNDERSCORE M E M O R Y UNDERSCORE A N A L Y T I C A L ;
+IN_MEMORY_TRANSACTIONAL : I N UNDERSCORE M E M O R Y UNDERSCORE T R A N S A C T I O N A L ;
```

Comment:
Just the question: don't we want to kepp the straigth line with the colons?

Reply:
But to answer your question, we failed to do so with transactional management keyword, updated it